src/cli_plugin/install/settings.js

```diff
@@ -37,10 +37,14 @@ import expiry from 'expiry-js';
 import { fromRoot } from '../../core/server/utils';
 
 function generateUrls({ version, plugin }) {
-  return [
-    plugin,
-    `https://artifacts.opensearch.org/downloads/kibana-plugins/${plugin}/${plugin}-${version}.zip`,
-  ];
+  return [plugin, generatePluginUrl(version, plugin)];
+}
+
+function generatePluginUrl(version, plugin) {
+  const baseUrl = 'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards';
```

Comment:
very minor: can this be a global const outside of the function so that it doesn't have to be created every time the function is called? I think most of the repo uses `SCREAMING_SNAKE_CASE` for those.

Reply:
And then if you do this we can just have a return without creating any variables.